airbyte-integrations/connectors/source-twilio/metadata.yaml

```diff
@@ -1,26 +1,24 @@
 data:
   allowedHosts:
     hosts:
-      - api.twilio.com
-      - monitor.twilio.com
-      - chat.twilio.com
-      - trunking.twilio.com
+      - TODO # Please change to the hostname of the source.
+  registries:
+    oss:
+      enabled: false
+    cloud:
+      enabled: false
   connectorSubtype: api
   connectorType: source
-  definitionId: b9dc6155-672e-42ea-b10d-9f1f1fb95ab1
-  dockerImageTag: 0.10.0
+  definitionId: 14dac961-bba0-482f-be0b-7462535467dd
+  dockerImageTag: 0.1.0
```

Comment:
Why was the definition id changed? Can you change it back? 

See above comments about versioning